templates/addons/aurora/cf.yml

```diff
@@ -140,7 +140,7 @@ Resources:
       TargetId: !Ref {{logicalIDSafe .ClusterName}}DBCluster
       TargetType: AWS::RDS::DBCluster
 Outputs:
```

Comment:
Do we not need to output the endpoint? https://github.com/copilot-example-voting-app/api/blob/main/copilot/api/addons/db.template.yaml#L116-L118

Reply:
Not really. The secret itself will contain host and port information, which is all it needs to access the db.

Secret manager stores all those information in the secret string automatically ([doc](https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage_create-basic-secret.html): "Choose the database instance from the list. Secrets Manager retrieves the connection details about the database by querying the chosen instance.")

Reply:
Neat! I didn't know that :D